index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="styles/styles.css" />
+  <script defer src="scripts/index.js"></script>
+  <title>Haker_news_JS</title>
+</head>
+<body>
+  <main class="page">
+    <header class="page__header top-panel">      
+      <h1 class="logo__title">Hacker News</h1>
+      </header>    
+      <section class="page__news">
+        <ol id="news-list" class="generic-list">
+        </ol>
+      </section>
+      <aside class="page__form">
+        <form class="form" name="user">
+          <h2 id="loginFormTitle" class="form__title">Login form</h2>
+            <div class="form__control field">
+            <label for="nameField" class="field__label">User name</label>
+            <input id="nameField" type="text" class="field__input" placeholder="name" name="name">
+          </div>
+          <div class="form__control field">
+            <label class="field__label">Gender
+            </label>
+            <div id="genderField" class="form__radio radio">
+              <input class="radio__input" type="radio" id="genderMale" name="gender" value="Male">
+              <label for="genderMale">Male</label>
+              <input type="radio" class="radio__input" id="genderFemale" name="gender" value="Female">
+              <label for="genderFemale">Female</label>
+            </div>
+          </div>
+          <div class="form__control field">
+            <label for="ageField" class="field__label">Age</label>
+            <input id="ageField" class="field__input" type="number" name="age">
+          </div>
+          <div class="form__control field">
+            <label for="select" class="field__label">Country </label>
+            <select id="select" class="field__input" name="country">
+              <option value=""></option>
+              <option value="Ukraine">Ukraine</option>
+              <option value="Poland">Poland</option>
+              <option value="Canada">Canada</option>
+              <option value="USA">USA</option>
+            </select>
+
+          </div>
+          <button id="submitBtn" class="form__button" type="submit">Відправити</button>
+        </form>
+        <article id="userInfo" class="user-info">
+          <h2 class="user-info__title">User</h2>
+          <div id="userInfoName" class="user-info__item"></div>
```

Comment:
not sure if ```class="user-info__item"``` is a good naming
how about ```class="user-info__label"``` ?

Reply:
Fixed